landoapi/api/try_push.py

```diff
@@ -82,18 +94,13 @@ def parse_revisions_from_request(
         io.BytesIO(convert_json_patch_to_bytes(patch)) for patch in patches
     )
 
+    patch_helpers = (patch_format.patch_helper(patch) for patch in patches_bytes)
+
     try:
-        if patch_format == PatchFormat.HgExport:
```

Comment:
It looks like `patch_format` is now unused?

Reply:
It's used to generated `patch_helpers` just above.

Reply:
Thanks, never mind!

Reply:
I think if `patch_helper` is moved outside the Enum it would make this also clearer, even though this is fuctional. I will bundle this along with the nit though, since I don't think it's a blocker.